pkg/web/api/v1/transaction.go

```diff
@@ -224,3 +235,126 @@ func NewSecureEnvelopeList(page *models.SecureEnvelopePage) (out *EnvelopesList,
 
 	return out, nil
 }
+
+func NewEnvelope(env *envelope.Envelope) (out *Envelope, err error) {
+	switch state := env.State(); state {
+	case envelope.Error:
+		return &Envelope{Error: env.Error()}, nil
+	case envelope.Clear:
```

Comment:
Does `envelope.Clear` mean that an envelope is empty?

Reply:
Never mind. I was a little confused, but now I understand where the states came from.